cli/src/languages.rs

```diff
@@ -147,6 +159,114 @@ pub async fn run(ad4m_client: Ad4mClient, command: Option<LanguageFunctions>) ->
             ad4m_client.languages.remove(address).await?;
             println!("Language removed");
         }
+        LanguageFunctions::GenerateBootstrap {
+            agent_path,
+            passphrase,
+            ad4m_host_path,
+            seed_proto,
+        } => {
+            println!(
+                "Attempting to generate a new bootstrap seed using ad4m-host path: {:?} and agent path: {:?}\n",
+                ad4m_host_path,
+                agent_path
+            );
+
+            //Warn the user about deleting ad4m agent during publishing
+            println!("WARNING... THIS WILL DELETE YOUR EXISTING AD4M AGENT AND REPLACE WITH SUPPLIED PUBLISHING AGENT, PLEASE BACKUP BEFORE PROCEEDING\n\n");
```

Comment:
I don't think this needs to be the case, no? We don't need to use `executor_data_path()` below, but could just use a temp directory.

Reply:
Ah yes I think that is possible and would avoid having to backup and delete existing agents.